### PR TITLE
Nova Scotia Age tax credit

### LINKED ISSUE #257: Nova Scotia Age tax credit
Reference: https://www.canada.ca/en/revenue-agency/services/forms-publications/tax-packages-years/general-income-tax-benefit-package/nova-scotia/5003-pc/information-residents-nova-scotia.html#P90-4569

### PR DESCRIPTION
**Pull request by ziyuliuzilla**
Fix #257

### CODE REVIEW

File: policyengine_canada/parameters/gov/provinces/ns/tax/income/credits/age_tax_credit/income_eligibility.yaml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+description: The taxable income eligibility for the Nova Scotia Age tax credit.
```
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/ns/tax/income/credits/age_tax_credit/income_eligibility.yaml:1]:
```suggestion
description: Nova Scotia limits its age tax credit to filers with taxable income below this amount.
```

File: policyengine_canada/parameters/gov/provinces/ns/tax/income/credits/age_tax_credit/age_eligibility.yaml
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/ns/tax/income/credits/age_tax_credit/age_eligibility.yaml:1]: folder can just be `age`

File: policyengine_canada/parameters/gov/provinces/ns/tax/income/credits/age_tax_credit/base.yaml
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/ns/tax/income/credits/age_tax_credit/base.yaml:1]: rename `amount`

File: policyengine_canada/parameters/gov/provinces/ns/tax/income/credits/age_tax_credit/base.yaml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+description: Nova Scotia provides this base amount under the Age tax credit.
```
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/ns/tax/income/credits/age_tax_credit/base.yaml:1]: Let's just use `base` when there are multiple elements
```suggestion
description: Nova Scotia provides this amount under the Age Tax Credit.
```

File: policyengine_canada/parameters/gov/provinces/ns/tax/income/credits/age_tax_credit/age_eligibility.yaml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+description: The age eligibility for the Nova Scotia Age tax credit.
```
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/ns/tax/income/credits/age_tax_credit/age_eligibility.yaml:1]:
```suggestion
description: Nova Scotia limits its Age Tax Credit to people this age or older.
```

File: changelog_entry.yaml
Comment (PavelMakarchuk) [inline changelog_entry.yaml:1]: run `make format`

File: policyengine_canada/parameters/gov/provinces/ns/tax/income/credits/age/income_eligibility.yaml [hunk 4]
```diff
@@ -0,0 +1,9 @@
+description: Nova Scotia limits its age tax credit to filers with taxable income below this amount.
+values:
+  2022-12-31: 24_000
+metadata:
+  unit: currency-CAD
+  label: Nova Scotia Age tax credit taxable income eligibility
```
Comment (MaxGhenis) [inline policyengine_canada/parameters/gov/provinces/ns/tax/income/credits/age/income_eligibility.yaml:6]:
```suggestion
  label: Nova Scotia Age tax credit taxable income limit
```

**Review by MaxGhenis — APPROVED** [review]

**Pull request merged**